tracer/src/Datadog.Trace/RemoteConfigurationManagement/RemoteConfigurationManager.Subscription.cs

```diff
@@ -0,0 +1,45 @@
+// <copyright file="RemoteConfigurationManager.Subscription.cs" company="Datadog">
```

Comment:
I find it a bit strange that this file contains the subscribe and unsubscribe but no invoke. Perhaps the invoke logic should be moved here?

Reply:
On my end, I'm not sure the partial class brings a lot of readability to be honest.  Maybe merge both?

Reply:
Fixed in 549eec7407f84c36068859253400159a48889a36 , the invoke happens from the subscription class also, and removed the partial class for subscription